sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -648,8 +649,8 @@
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public PagedFlux<KeyBase> listKeys() {
```

Comment:
We will still need a package-private listKeys(Context) API, that the sync APIs can call into, correct?

Reply:
Yes, all APIs of Keys and Secrets async clients will need a package-private method with context to support sync APIs. I will add those in a separate PR before @samvaity starts on KeyVault.

Reply:
I have included package-private methods in this PR itself.